Other/MergeShops/Adam1a1Merge.cs

```diff
@@ -0,0 +1,239 @@
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Fresh Ectoplasm":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8009);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("vendorbooths", "Caffeine Imp", "Coffee Beans", 10);
+                        Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10);
+                        Core.HuntMonster("charredpath", "Toxic Wisteria", "Necessary Antidote");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "IOU Slip":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8009);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("vendorbooths", "Caffeine Imp", "Coffee Beans", 10);
+                        Core.HuntMonster("djinn", "Lamia", "Tasty Poison", 10);
+                        Core.HuntMonster("charredpath", "Toxic Wisteria", "Necessary Antidote");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "EctoBlade":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("shadowfallwar", "Skeletal Fire Mage", "EctoBlade", isTemp: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Ectoplasmic Chains":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    Core.RegisterQuests(8010);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("temple", "Doomwood Ectomancer", "Refined Ectoplasm", 10);
+                        Core.HuntMonster("ectocave", "Ektorax", "Ektorax's Ectoplasm");
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Bongo Cart Pet":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Farm);
+                    if (!Bot.Quests.IsUnlocked(8013)) {
```

Comment:
story checks dont require a if() check, just the story line, as the script that your referenceing has its own checks

Reply:
once again would go with the other above Line 45